### PR TITLE
38 clarify upload benchmark issues

### LINKED ISSUE #38: Clarify issues surrounding automatic upload of benchmark results
For the time being, calling the script which uploads benchmark results using the `postscripthook` in the Iguana `config.yml` results in strange and unpredictable behaviour. This issue aims to clarify the behaviour in the `README.md` as well as improve on some other areas in the codebase related to this script.

### PR DESCRIPTION
**Pull request by doogyb**
Script to upload the ntriples to a triplestore to allow for querying the benchmark metrics.

closes #38

### CODE REVIEW

File: src/node_modules/queries/util.mjs [hunk 1]
```diff
@@ -1,17 +1,18 @@
 import { promises as fs } from 'fs'
+import * as path from 'path'
```
Comment (mindrones) [inline src/node_modules/queries/util.mjs]: We usually use:

```suggestion
import path from 'path'
```

File: src/bin/benchmarking/uploadResults.mjs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+.requiredOption(
+	'-p, --path <path>',
+	'Path to directory benchmark results'
+);
+
+program.parse(process.argv);
+const options = program.opts();
+
+(async () => {
+	// remove http:// from string
```
Comment (mindrones) [inline src/bin/benchmarking/uploadResults.mjs]: I'd prefer to declare a `main` and invoking it explicitly

File: src/benchmarking/config.yml [hunk 3]
```diff
@@ -1,20 +1,18 @@
...
 #Your connection
 connections:
-  - name: 'Virtuoso'
-    endpoint: 'http://ec2-18-170-45-162.eu-west-2.compute.amazonaws.com:8890/sparql'
-  - name: 'Neptune'
-    endpoint: 'https://geonames-rdf-01.cluster-ro-ci9272ypsbyf.eu-west-2.neptune.amazonaws.com:8182/sparql'
+  - name: "Virtuoso"
+    endpoint: "http://ec2-18-170-45-162.eu-west-2.compute.amazonaws.com:8890/sparql"
+  - name: "Neptune"
+    endpoint: "http://ec2-18-170-45-162.eu-west-2.compute.amazonaws.com:3000/sparql"
```
Comment (mindrones) [inline src/benchmarking/config.yml]: Is this duplicate of info we have in `src/node_modules/config.mjs`?

File: src/bin/benchmarking/uploadResults.mjs (hunk 2 above)
Comment (mindrones) [inline src/bin/benchmarking/uploadResults.mjs]: These things could be precomputed in a file like `src/node_modules/configured.mjs` or so, that would import from `src/node_modules/config.mjs`, with everyone importing from one and/or the other depending on the circumstances.

This way we'd have computation happenning only once and in a single place.

**Review by mindrones — CHANGES_REQUESTED** [review]
first pass

File: src/benchmarking/config.yml (hunk 3 above)
Comment (doogyb) [reply src/benchmarking/config.yml]: Yes, but seeing as the Iguana tool imports this directly, I'm not sure how to merge the `config.mjs` with `config.yml`... Unless we generate this file dynamically? Which seems overkill. I could be overengineering this though

**Review by mindrones — APPROVED** [review]
GTM apart from some double quotes here and there, that I'd change with single quotes

**Pull request merged**